Documentation/benchmarks/etcd-2-2-0-rc-benchmarks.md

```diff
@@ -1,4 +1,8 @@
-## Physical machines
+---
+title: Benchmarking etcd v2.2.0-rc
+---
+
+## Physical machine
 
```

Comment:
I notice that in some places you replace the markdown title, moving it below the metadata and others you don't. Is this intentional? Feels like we still want the original ## title but maybe it is redundant?

Reply:
In this case, I do think that "Benchmarking etcd v2.2.0-rc" is the intended title based on the other docs in that directory

Reply:
In general what I'm doing is performing a delicate dance whereby I'm making sure that this repo is "well formed" for consumption by the website repo while also making sure that the website repo is able to successfully build the docs here. So it will likely take a handful of PRs here to get that right. Apologies for any technical difficulties 😄